README.md

```diff
@@ -1,2 +1,159 @@
 # linux-prewarm-process-rfc
 WIP - For now it's just an idea of how to launch processes ahead of time for bash scripts
+
+# context / problem
+shell scripts (in any flavour/variant) are incredibly composable due to unix pipelines. 
```

Comment:
Another idea (in this space) is a project like nushell, which aimed to re-implement most of these functions but (as you note) in an efficient / faster way. And nushell is made for combining commands / doing programmatic stuff directly in the console. https://www.nushell.sh/

Reply:
That's an interesting tool. I haven't heard of it yet (thanks for sharing :)).\

Based on a cursory reading of the nushell source code, it feels like their value proposition is more towards data being typed which I think does speed up the development time but it still faces the same 'issues' that we are trying to solve given that it run executables in the same way that bash does